WalletWasabi.Fluent/Controls/ClipboardCopyButton.axaml.cs

```diff
@@ -37,6 +37,7 @@ private async Task CopyToClipboardAsync()
 		if (Application.Current is { Clipboard: { } clipboard })
 		{
 			await clipboard.SetTextAsync(Text);
+			await Task.Delay(1000);
```

Comment:
Why the delay?

Reply:
Added this comment in the code to clarify its purpose:

```
await Task.Delay(1000); // Introduces a delay while the animation is playing (1s). This will make the command 'busy' while being animated, avoiding reentrancy.
```

Indirectly, this will make CopyableItem able to "detect" when the animation has stopped playing in order to hide the button when it's appropriate.